eahub/base/views.py

```diff
@@ -172,11 +172,13 @@ def form_valid(self, form):
 def healthCheck(request):
     return HttpResponse(status=204)
 
-
 def page_not_found(request, exception):
     if not isinstance(exception, exceptions.Quiet404):
         mail.mail_managers(
             "Broken link",
             loader.render_to_string("emails/broken_link.txt", {"request": request}),
         )
     return defaults.page_not_found(request, exception)
+
+def gone(request):
+    return HttpResponse(status=410)
```

Comment:
This just returns a blank page to the user, which I don't think we want. They should see a page built from our templates, with an explanation of why the specific thing they were looking for doesn't exist anymore.

Also, your editor still isn't handling end-of-file newlines properly.